modules/administration/pages/space-management.adoc

```diff
@@ -7,6 +7,8 @@ Running out of disk space can have a severe impact on the {productname} database
 You can modify which directories are monitored, and the warnings that are created.
 All settings are configured in the [path]``/etc/rhn/rhn.conf`` configuration file.
 
+When the available space in one of these monitored directories falls below a warning threshold, an email is sent to the configured email address and a message is shown at the top of the login page.
```

Comment:
```suggestion
When the available space in one of the monitored directories falls below a warning threshold, a message is sent to the configured email address and a notofication is shown at the top of the sign-in page.
```

Reply:
There is a typo ("notofication") in your suggested change. If you edit it, I will commit it together with the other one.